mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,395 @@
+        self.drop_rate = drop_rate
+        if self.has_shortcut and drop_rate > 0:
+            self.dropout = DropPath(drop_rate)
+        else:
+            self.dropout = nn.Identity()
+
+    def forward(self, x: Tensor) -> Tensor:
+        result = self.expand_conv(x)
+        result = self.dwconv(result)
+        result = self.se(result)
+        result = self.project_conv(result)
+
+        if self.has_shortcut:
+            if self.drop_rate > 0:
+                result = self.dropout(result)
+            result += x
+
+        return result
+
+
+class FusedMBConv(nn.Module):
+
+    def __init__(self, kernel_size: int, input_c: int, out_c: int,
+                 expand_ratio: int, stride: int, se_ratio: float,
+                 drop_rate: float, norm_layer: Callable[..., nn.Module]):
+        super(FusedMBConv, self).__init__()
+
+        assert stride in [1, 2]
+        assert se_ratio == 0
+
+        self.has_shortcut = stride == 1 and input_c == out_c
+        self.drop_rate = drop_rate
+
+        self.has_expansion = expand_ratio != 1
+
+        activation_layer = nn.SiLU  # alias Swish
+        expanded_c = input_c * expand_ratio
+
+        # 只有当expand ratio不等于1时才有expand conv
+        if self.has_expansion:
+            # Expansion convolution
+            self.expand_conv = ConvBNAct(
+                input_c,
+                expanded_c,
+                kernel_size=kernel_size,
+                stride=stride,
+                norm_layer=norm_layer,
+                activation_layer=activation_layer)
+
+            self.project_conv = ConvBNAct(
+                expanded_c,
+                out_c,
+                kernel_size=1,
+                norm_layer=norm_layer,
+                activation_layer=nn.Identity)  # 注意没有激活函数
+        else:
+            # 当只有project_conv时的情况
+            self.project_conv = ConvBNAct(
+                input_c,
+                out_c,
+                kernel_size=kernel_size,
+                stride=stride,
+                norm_layer=norm_layer,
+                activation_layer=activation_layer)  # 注意有激活函数
+
+        self.out_channels = out_c
+
+        # 只有在使用shortcut连接时才使用dropout层
+        self.drop_rate = drop_rate
+        if self.has_shortcut and drop_rate > 0:
+            self.dropout = DropPath(drop_rate)
+        else:
+            self.dropout = nn.Identity()
+
+    def forward(self, x: Tensor) -> Tensor:
+        if self.has_expansion:
+            result = self.expand_conv(x)
+            result = self.project_conv(result)
+        else:
+            result = self.project_conv(x)
+
+        if self.has_shortcut:
+            if self.drop_rate > 0:
+                result = self.dropout(result)
+
+            result += x
+
+        return result
+
+
+@MODELS.register_module()
+class EfficientNetV2(BaseBackbone):
+    # repeat, kernel, stride, expansion, in_c, out_c, se_ratio, operator
+    arch_settings = {
+        'base': [[1, 3, 1, 1, 16, 16, 0, 0], [2, 3, 2, 4, 16, 32, 0, 0],
```

Comment:
Could this algin with the efficientnetv1? https://github.com/open-mmlab/mmclassification/blob/2495400a98a3da9bd8cfb3731a711f4c77cc56a3/mmcls/models/backbones/efficientnet.py#L181

Reply:
For b0/b1/b2/b3 , the arch can be like  efficientnetv1, but for s/m/l/xl , timm and tf give a real arch_setting parameters , not  a ratio , so it may be a little difficult to algin with the efficientnetv1, i will try.